index.html

```diff
@@ -208,6 +208,31 @@ <h3>GetUserMediaSemantics enum</h3>
         </table>
       </div>
     </section>
+    <section id="mediastreamconstraints-dictionary-extensions">
+      <h3>MediaTrackConstraintSet dictionary extensions</h3>
+      <div>
+        <pre class="idl"
+>partial dictionary MediaTrackConstraintSet {
+  ConstrainBoolean efficientPixelFormat;
+};</pre>
+        <section>
+          <h2>Dictionary {{MediaTrackConstraintSet}} Members</h2>
+          <dl data-link-for="MediaTrackConstraintSet" data-dfn-for=
+          "MediaTrackConstraintSet" class="dictionary-members">
+            <dt><dfn><code>efficientPixelFormat</code></dfn> of type <span class=
```

Comment:
I would suggest s/efficientPixelFormat/powerEfficient/ to be more like https://www.w3.org/TR/media-capabilities/#media-capabilities-info.